<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elektronik Komponent Kayıt Kontrol Sistemi</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>buton_resim/anakart.png</normaloff>buton_resim/anakart.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="direnc_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Direnç
Hesapla</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/direnc.png</normaloff>buton_resim/direnc.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bobin_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bobin
Hesapla</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/bobin.png</normaloff>buton_resim/bobin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="kondansator_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kondansatör
Hesapla</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/kondansator.png</normaloff>buton_resim/kondansator.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="komponent_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Komponent
Kayıt-Kontrol</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/anakart.png</normaloff>buton_resim/anakart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cikis_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/cikis.png</normaloff>buton_resim/cikis.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Furore</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elektronik Komponent Kayit Kontrol Sistemi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Furore</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ender Yilmaz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Furore</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>171602012</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
